management/ssl_certificates.py

```diff
@@ -249,6 +250,7 @@ def rdata__str__(r):
 				s = r.to_text()
 				if isinstance(s, bytes):
 					 s = s.decode('utf-8')
+				s = str(ipaddress.ip_address(s))
```

Comment:
Ergh, sorry, I misread the code here and misunderstood what it was doing. Nevermind.

(But the overall comment I left still stands.)